modular_skyrat/master_files/code/datums/traits/negative.dm

```diff
@@ -8,6 +8,26 @@
 	medical_record_text = "Patient is incapable of communicating their emotions."
 	icon = "question-circle"
 
+/datum/quirk/fragile
+	name = "Fragility"
+	desc = "You feel incredibly fragile. Incoming brute and burn damage is increased!"
+	value = -6
+	mob_trait = TRAIT_FRAGILE
+	medical_record_text = "Patient's body has adapted to low gravity. Sadly low-gravity environments are not conducive to strong bone development."
+	icon = "question-circle" //placeholder
+
+/datum/quirk/fragile/post_add()
+	. = ..()
+	var/mob/living/carbon/human/H = quirk_holder
```

Comment:
To avoid using a single letter variable try changing it from `var/mob/living/carbon/human/H` to something more descriptive like `var/mob/living/carbon/human/user`, you'll also have to replace the instances of h in your code with the new variable

Reply:
Gotcha, thanks for explaining! Fixed in cbeea85